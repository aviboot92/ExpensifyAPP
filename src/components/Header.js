import React from 'react';
import { NavLink } from 'react-router-dom';


const Header = () => (
    <header>
        <h1>Expensify APP</h1>
        <NavLink activeClassName='is-active' exact to='/'>Home</NavLink>
        <NavLink activeClassName='is-active' to='/create'>Add Expense</NavLink>
        <NavLink activeClassName='is-active' to='/help'>Help</NavLink>
    </header>
);

export default Header;