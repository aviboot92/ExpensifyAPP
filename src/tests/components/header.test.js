import React from 'react';
// import ReactShallowRenderer from 'react-test-renderer/shallow';
import toJSON from 'enzyme-to-json';
import {shallow} from 'enzyme';
import Header from '../../components/Header';
import expectExport from 'expect';

test('should render header correctly', () => {
    const wrapper = shallow(<Header />);
    expect(toJSON(wrapper)).toMatchSnapshot();
    // const renderer = new ReactShallowRenderer();
    // renderer.render(<Header />);
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
    // expect(wrapper.find('h1').text()).toBe('Expensify APP');
})