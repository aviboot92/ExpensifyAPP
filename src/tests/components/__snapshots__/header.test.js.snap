// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<header>
  <h1>
    Expensify APP
  </h1>
  <ForwardRef
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Home
  </ForwardRef>
  <ForwardRef
    activeClassName="is-active"
    to="/create"
  >
    Add Expense
  </ForwardRef>
  <ForwardRef
    activeClassName="is-active"
    to="/help"
  >
    Help
  </ForwardRef>
</header>
`;
